qa/rpc-tests/sc_csw_nullifier.py

```diff
@@ -38,29 +42,40 @@ def setup_network(self, split=False):
                                  extra_args=[["-sccoinsmaturity=0", '-scproofqueuesize=0', '-logtimemicros=1', '-debug=sc', '-debug=py',
                                               '-debug=mempool', '-debug=net', '-debug=bench']] * NUMB_OF_NODES)
 
-        if not split:
-            # 1 and 2 are joint only if split==false
-            connect_nodes_bi(self.nodes, 1, 2)
-            sync_blocks(self.nodes[1:3])
-            sync_mempools(self.nodes[1:3])
-
-        connect_nodes_bi(self.nodes, 0, 1)
-        self.is_network_split = split
+        for idx in range(NUMB_OF_NODES - 1):
+            connect_nodes_bi(self.nodes, idx, idx+1)
+        self.is_network_split = False
         self.sync_all()
 
     def split_network(self):
-        # Split the network of three nodes into nodes 0-1 and 2.
-        assert not self.is_network_split
-        disconnect_nodes(self.nodes[1], 2)
-        disconnect_nodes(self.nodes[2], 1)
+        # Disconnect the "double spend" node from the network
```

Comment:
This comment should be # Disconnect the "to be isolated" node from the network (probably a copy-paste from `mempool_double_spend.py`.

Reply:
Yes, corrected